Initialtask/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.3"
+services:
+  mysql:
+    image: mysql:5.7
+    hostname: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
```

Comment:
do not pass credentials into docker-compose.yml and env file
User should set them manually or using setup script